index.html

```diff
@@ -446,20 +458,15 @@ <h5 data-i18n="footer.in-touch.header">Keep in Touch</h5>
                 }
 
                 // update button
-                if (OS != "OTHER" && buildNum) {
+                if (OS !== "OTHER" && buildNum) {
                     var tag = buildName.toLowerCase().split(" ").join("-"),
                         url = "https://github.com/adobe/brackets/releases/" + tag;
+
                     if (ext) {
                         url = "https://github.com/adobe/brackets/releases/download/" + tag + "/Brackets." + buildName.split(" ").join(".") + ext;
                     }
 
-                    ///if(OS=="WIN") url = "http://bit.ly/QrVHNc"; // (Temporary) TRACKING WIN
-                    ///else if(OS=="OSX") url = "http://bit.ly/1hQGEsh"; // (Temporary) TRACKING OSX
-
-                    var version_label = buildName.match(/(\d+.*$)/)[1];
-                    //if (OS != "UNKNOWN") version_label += " (" + OS + ")";
-                    $("#download-brackets-version").text(version_label);
-
+                    $("#download-brackets-version").text(buildNum);
```

Comment:
I was a bit puzzled by this change, but @MarcelGerber did the same in https://github.com/adobe/brackets.io/pull/139 so I'm good with it.